test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -229,6 +229,7 @@ def _test_code_common(
         inputs,
         include_ops,
         exclude_ops,
+        num_include_ops=None,
```

Comment:
nit: the new added parameter should be at the last?

Reply:
Oh. Thanks. I thought it would be better to be close to `include_ops`. Will refine.